xsd/netex-nl.xsd

```diff
@@ -21,13 +21,18 @@
 <!--                      Line.ShortName toegevoegd t.b.v. 'lijnleveringen'          -->
 <!--   v9.2.2  (jun.2021) versie v9.2.1 hernoemd tot v9.2.2                          -->
 <!--   v9.2.3  (jul.2021) toevoegen flexvervoer                                      -->
+<!--   v9.2.x  (okt.2022) Organsation is ofwel Operator of Authority				 -->
+<!--					  Overeenkomstig EU XSD 						             -->
 <!--                                                                                 -->
 <!-- Copyright:                                                                      -->
 <!--     Dit document is eigendom van Platform BISON onder Samenwerkingsverband DOVA -->
 <!--     en wordt gepubliceerd onder de Creative Commons Licentie by-nd/3.0/nl,      -->
 <!--     zie http://creativecommons.org/licences/by-nd/3.0/nl                        -->
 <!-- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -->
-<xsd:schema xmlns="http://www.netex.org.uk/netex" xmlns:netex="http://www.netex.org.uk/netex" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:gml="http://www.opengis.net/gml/3.2" targetNamespace="http://www.netex.org.uk/netex" elementFormDefault="qualified" attributeFormDefault="qualified" version="9.2.3" id="netex-nl-xsd">
+<xsd:schema xmlns="http://www.netex.org.uk/netex" xmlns:netex="http://www.netex.org.uk/netex"
+	xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:gml="http://www.opengis.net/gml/3.2"
+	targetNamespace="http://www.netex.org.uk/netex" elementFormDefault="qualified"
+	attributeFormDefault="qualified" version="9.2.3" id="netex-nl-xsd">
```

Comment:
```suggestion
<xsd:schema xmlns="http://www.netex.org.uk/netex" xmlns:netex="http://www.netex.org.uk/netex" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:gml="http://www.opengis.net/gml/3.2" targetNamespace="http://www.netex.org.uk/netex" elementFormDefault="qualified" attributeFormDefault="qualified" version="9.2.3" id="netex-nl-xsd">
```

Reply:
Doe dit soort wijzigingen vooral, maar dan allemaal in een losse pull request. Zodat er semantisch niets tegelijkertijd veranderd.